src/modules/events/components/EventListNavigation.tsx

```diff
@@ -34,13 +38,22 @@ export const EventListNavigation = observer(() => {
 
 	return (
 		<div className='events-nav'>
-			<button className='button-base' onClick={getPrevEvents}>
+			<Chip
+				className='events-nav__arrow-wrapper events-nav__arrow-wrapper--left'
+				onClick={getPrevEvents}>
+				<EventsArrowIcon />
+			</Chip>
+			<button className='button-base button-base--events' onClick={getPrevEvents}>
 				Show previous
 			</button>
 			<span className='events-nav__timestamp'>{formatTimestamp(timestamp)}</span>
-			<button className='button-base' onClick={getNextEvents}>
+			<button className='button-base button-base--events' onClick={getNextEvents}>
```

Comment:
I think it's better to move chip inside the button instead of having 2 html elements with onClick handler, one of which isnt button